specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2025-06-01-preview/sqlServerInstances.json

```diff
@@ -0,0 +1,2084 @@
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        }
+      }
+    },
+    "SqlServerInstanceManagedInstanceLinkAssessment": {
+      "description": "The status of an individual requirement for MI Link.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the requirement."
+        },
+        "category": {
+          "$ref": "#/definitions/MiLinkAssessmentCategory"
+        },
+        "status": {
+          "type": "string",
+          "description": "The status of the requirement.",
+          "enum": [
+            "Success",
+            "Warning",
+            "Failure"
+          ],
+          "x-ms-enum": {
+            "name": "assessmentStatus",
+            "modelAsString": true
+          }
+        },
+        "information": {
+          "type": "string",
+          "description": "The information text about requirement."
+        },
+        "additionalInformation": {
+          "type": "string",
+          "description": "Additional details about the check, including any recommended actions."
+        },
+        "failingDbs": {
+          "type": "array",
+          "description": "List of names of databases that are failing the given assessment.",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Authentication": {
+      "description": "Authentication related configuration for the SQL Server Instance.",
+      "type": "object",
+      "properties": {
+        "mode": {
+          "type": "string",
+          "description": "Mode of authentication in SqlServer.",
+          "enum": [
+            "Mixed",
+            "Windows",
+            "Undefined"
+          ],
+          "x-ms-enum": {
+            "name": "mode",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Mixed",
+                "description": "Mixed mode authentication for SQL Server which includes windows and SQL Authentication."
+              },
+              {
+                "value": "Windows",
+                "description": "Windows Authentication for SQL Server."
+              },
+              {
+                "value": "Undefined",
+                "description": "Used for scenarios were the mode cannot be determined."
+              }
+            ]
+          }
+        },
+        "sqlServerEntraIdentity": {
+          "type": "array",
+          "description": "Entra Authentication configuration for the SQL Server Instance.",
+          "items": {
+            "$ref": "./common.json#/definitions/EntraAuthentication"
+          }
+        }
+      }
+    },
+    "Schedule": {
+      "description": "The scheduling configuration.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Indicates whether scheduling is enabled."
+        },
```

Comment:
Enums are highly recommended over booleans. 

Please model this as an enum to allow for multiple values and future proofing.